templates/cinder/config/10-cinder_wsgi.conf

```diff
@@ -3,16 +3,18 @@
 <VirtualHost *:8776>
   ServerName {{ $vhost.ServerName }}
 
-  ## Vhost docroot
-  DocumentRoot "/var/www/cgi-bin/cinder"
-
-  ## Directories, there should at least be a declaration for /var/www/cgi-bin/cinder
-
-  <Directory "/var/www/cgi-bin/cinder">
-    Options -Indexes +FollowSymLinks +MultiViews
-    AllowOverride None
-    Require all granted
-  </Directory>
+  ## Set DocumentRoot and Directory permissions only if the path exists
+  <If "-d '/var/www/cgi-bin/cinder'">
+    ## Vhost docroot
+    DocumentRoot "/var/www/cgi-bin/cinder"
```

Comment:
-1:  this will fail with the following:

```
+ exec /usr/sbin/httpd -DFOREGROUND
AH00526: Syntax error on line 9 of /etc/httpd/conf.d/10-cinder_wsgi.conf:
DocumentRoot not allowed in <If> context
```

I think we need to set `DocumentRoot` anyway, outside the `If` condition.